plugins/module_utils/utils.py

```diff
@@ -569,6 +569,17 @@ def trigger_restart_operation(idrac, restart_type="GracefulRestart", resource_id
     return resp, error_msg
 
 
+def power_on_operation(idrac, restart_type="On", resource_id=None):
```

Comment:
There's already function to restart the idrac, we can utilize the same. This function is duplicate of trigger_restart_operation.

Reply:
Done